pkg/virt-handler/container-disk/mount.go

```diff
@@ -354,7 +355,7 @@ func (m *mounter) Unmount(vmi *v1.VirtualMachineInstance) error {
 		log.DefaultLogger().Object(vmi).Infof("Looking to see if containerdisk is mounted at path %s", entry.TargetFile)
 		file, err := safepath.NewFileNoFollow(entry.TargetFile)
 		if err != nil {
-			if os.IsNotExist(err) {
+			if errors.Is(err, os.ErrNotExist) {
```

Comment:
There is one more such `os.IsNotExist(err)` check on the line `476` in this file:

https://github.com/kubevirt/kubevirt/blob/b862288214babf12b06ce808efaf2adbf592e4a1/pkg/virt-handler/container-disk/mount.go#L476

Probably should also be replaced with `errors.Is(err, os.ErrNotExist)`